library/Makefile

```diff
@@ -37,7 +37,7 @@ endif
 
 SOEXT_TLS=so.13
 SOEXT_X509=so.1
-SOEXT_CRYPTO=so.4
+SOEXT_CRYPTO=so.3
```

Comment:
Blocker: bad merge/rebase: this commit should not touch version numbers

Reply:
Thanks for spotting.